web/ui/react-app/src/App.css

```diff
@@ -156,6 +156,7 @@ button.execute-btn {
   align-items: center;
   padding: 0 5px;
   border-radius: 3px;
+  flex-wrap: wrap;
 }
 
```

Comment:
```suggestion

.legend-item > div {
  flex-wrap: wrap;
}
```

With this changing `Legend.tsx` will give fixing.
![Screenshot from 2020-02-11 01-10-08](https://user-images.githubusercontent.com/42701709/74183604-4de0a100-4c6b-11ea-8191-6c4ff344fa93.png)
### Result before change

![Screenshot from 2020-02-11 01-12-03](https://user-images.githubusercontent.com/42701709/74183765-96985a00-4c6b-11ea-9a67-7083ef412e0c.png)
### Result after change
![Screenshot from 2020-02-11 01-08-37](https://user-images.githubusercontent.com/42701709/74183504-14a83100-4c6b-11ea-87bb-d57b02c2577f.png)

Reply:
Yeah, thanks! I tried this as well but decided to leave it as I propose it in the issue https://github.com/prometheus/prometheus/issues/6582#issuecomment-572720891
But in case this would be the desired result I would put the div in the component

Reply:
@drumilpatel2000 just saw that initially, I'm wrapping all the spans in a div in order to align the second row with the Series name...so thanks for the reminder.

Reply:
Seems okay for me :+1:

Reply:
@juliusv @roidelapluie please, review when you have a chance.